src/index.ts

```diff
@@ -0,0 +1,8 @@
+export {
+  type DscvrClientProtocol,
+  type DscvrFramesRequest,
+  type DscvrUntrustedData,
+  type DscvrValidationResponse,
+  dscvrClientProtocolPrefix,
```

Comment:
I suggest moving `dscvrClientProtocolPrefix` to `src/constants.ts` instead of `src/types.ts` as it's an `const` that is being exported rather than a `type`

Reply:
Done